js/react/components/nav/index.jsx

```diff
@@ -0,0 +1,154 @@
+import PropTypes from 'prop-types';
+
+function Nav(props) {
```

Comment:
please try to create a new navbar with this in mind 
https://getbootstrap.com/docs/5.3/components/navbar/#responsive-behaviors

right now we have two separate ones but bootstrap is able to use only one navbar and collapse it on mobile